modin/test/storage_formats/omnisci/test_internals.py

```diff
@@ -0,0 +1,88 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+import pytest
+import sys
+import subprocess
+
+
+@pytest.mark.parametrize(
+    "import_strategy",
+    [
+        pytest.param(
+            """
+import modin.config as cfg
+cfg.Engine.put('Native') # 'omniscidbe'/'dbe' would be imported with dlopen flags first time
+cfg.StorageFormat.put('Omnisci')
+cfg.IsExperimental.put(True)
+import modin.pandas as pd
+""",
+            id="config_first-import_second",
+        ),
+        pytest.param(
+            """
+import modin.pandas as pd
+import modin.config as cfg
+cfg.Engine.put('Native')
+cfg.StorageFormat.put('Omnisci')
+cfg.IsExperimental.put(True)
```

Comment:
The case when we import pyarrow.gandiva prior omnisci should work so this should be removed.